main/src/ca/uwaterloo/flix/language/phase/unification/set/SetFormula.scala

```diff
@@ -56,6 +56,20 @@ sealed trait SetFormula {
       SortedSet.from(varsPos.map(_.x)) ++ varsNeg.map(_.x) ++ other.flatMap(_.variables)
   }
 
+  /** Faster alternative to `this.variables.contains(v)`. */
```

Comment:
Faster in practice?

Reply:
Ignoring compiler magic, yes. You 1) don't have to build the variable set and 2) you can early exit